requirements.txt

```diff
@@ -0,0 +1,108 @@
+activity-log-manager==0.8.0
```

Comment:
Looks like you regenerated requirements.txt in root dir, instead of api/. Delete this file.

If you want, you can change api/README.md to work from root dir instead of api/dir.

Reply:
Done